internal/glance/static/js/todo.js

```diff
@@ -116,7 +113,7 @@ function Todo(id) {
     const saveItems = () => {
         if (isDragging) return;
 
-        saveToLocalStorage(
+        storage.save(
             id, items.children.map(item => item.component.serialize())
         );
```

Comment:
We should somehow handle any error in this, after the return issue in the widget-storage.js file has been resolved. Even if it's just an alert. It's not fancy, but to my knowledge this frontend does not provide any notification functionality, like toasts, at the moment.

(Implementing such a notification functionality would be out of the scope of this PR and should be implemented in another PR, imho)